src/components/Files/LinkToSong/LinkToSong.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import SongIcon from '../../SongIcon';
+import HeartFavorite from '../../HeartFavorite';
+import ContextMenu from '../../ContextMenu';
+
+const LinkToSongComponent = ({
+  fileName, isPlaying, selectSong, path,
+}) => {
+  const isPlayingClassName = isPlaying ? 'is-playing' : '';
+
+  return (
+    <div
+      onClick={selectSong}
```

Comment:
`role="button"` is a nice addition, however, I personally hate adding these tags to make a clickable div compliant. I rather have an actual button instead. Food for thought.